src/Ztm.WebApi/ServicesExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
+using Ztm.Zcoin.NBitcoin.Exodus;
+
+namespace Ztm.WebApi
+{
+    public static class ServicesExtensions
```

Comment:
This one should go to `Ztm.Zcoin.NBitcoin` instead.

Reply:
moved